src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Index.cshtml

```diff
@@ -184,10 +184,7 @@
                             <div class="tab-pane" id="protocol-help">
                                 <ol>
                                     <li>
-                                        Generate json file using Protocol parser
-                                    </li>
-                                    <li>
-                                        Upload the resulting json file.
+                                        Upload generated yml file from autorest 
```

Comment:
QQ, why don't we upload an autorest readme md instead?

Reply:
And run autorest as part of the generation?

Reply:
That's the next step. These are interim solution while Libba is working on parser. Our final goal is to generate yml file within APIView itself.

Reply:
or have a UI where you can point to one or multiple spec URLs